chore: config faststore ui (#1495)
vtex/faststore: apps/docs/docs/reference/ui/atoms/Label.mdx

## Customization

> For futher customization, use the data-attr below:

Reviewer: ```suggestion
For further customization, you can use the following data attributes:
```
